pyrft/random_field_generation.py

```diff
@@ -139,13 +144,15 @@ def wfield(mask, fibersize, field_type = 'N', field_params = 3):
 
   Examples
   ---------------------
-  exF = pr.wfield(1, 10)
-  exF = pr.wfield((5,5), 10)
+example_field = pr.wfield(15, 10); print(example_field)

```

Comment:
one command per line for readability

Reply:
Rather than pointing out every instance where this occurs, I think it may be better for us to discuss these things. I think we have some stylistic coding differences. I feel like what happens within the examples section should be treated differently than the main code since the main purposes of examples is to test the function and to serve as copy paste.

Reply:
But this code is in the main library, not the in the examples section.

Again, having a debate on this is as useless as discussing whether you should drive on the left or the right side of the road: you simply need to accept local conventions. 
There is one aspect that you may not yet be aware of: that we can do automatic documentation generation with sphinx, but for this you need all the docstrings to be well-formatted

Reply:
I just realised that I can copy paste the examples from the examples section in a different way even when there is an indent so I'm now happy to make these changes.

In general I'm happy to change things to conform to pep8, but these issues don't show up on my linter, hence my confusion. Maybe what happens within the examples section is not picked up by the linter?

I agree that it a good idea to conform to the community standard.

Reply:
And yeah I'm not aware of the sphinx doc generation but sounds useful/interesting!